Bottom: 0da414a7f4a910f56f31861256f160c57d24c8db
Top:    6336e8d5672dab7c7b5b86d02e0b7450b2562e4e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:08 +0000

Copy contour is_round information when copying polygons


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 364d472..039b9b5 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3478,6 +3478,10 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
   (*dst)->xmin = src->xmin, (*dst)->xmax = src->xmax;
   (*dst)->ymin = src->ymin, (*dst)->ymax = src->ymax;
   (*dst)->area = src->area;
+  (*dst)->is_round = src->is_round;
+  (*dst)->cx = src->cx;
+  (*dst)->cy = src->cy;
+  (*dst)->radius = src->radius;
 
   for (cur = NEXT_EDGE (&src->head); cur != &src->head; cur = NEXT_VERTEX (cur))
     {
